### ISSUE TITLE
send_request() got an unexpected keyword argument 'verbose'

### ISSUE THREAD

**Issue by archtaurus** [cpburnz/python-amazon-mws#4]
calling amazonmws.products.MWSProducts methods it got:
```
    196                         raise ValueError("id_type:{!r} is not 'ASIN' or 'SellerSKU'.".format(id_type))
    197 
--> 198                 return self.send_request(args, path=self.path, verbose=verbose)
    199 
    200         def get_competitive_pricing(self, marketplace_id, id_type, id_list, verbose=None):

TypeError: send_request() got an unexpected keyword argument 'verbose'
```

**Comment by archtaurus** [comment]
I noticed get_products function says ACTION name is "get_products_for_id", limit is 5 items per request.
In latest dom of mws I think it is now called "GetMatchingProduct", and limit is 10 items per request.

Does this mean your code is out of date while some of them is still working and some is not?

**Comment by cpburnz** [comment]
I fixed the issue. There was a mismatch between name of the debug argument.

> Is this package still works? I tried it and I think it is nice. But is it still working?

> Does this mean your code is out of date while some of them is still working and some is not?

Honestly, I haven't used this in years so it depends on if Amazon has updated their API. The Products API this library was made for was version 2011-10-01 and the version on their website says the same version so it should work (excluding any bugs like what you found).

> I noticed get_products function says ACTION name is "get_products_for_id", limit is 5 items per request.
In latest doc of mws I think it is now called "GetMatchingProduct", and limit is 10 items per request.

It looks like *GetMatchingProduct* is just for ASINs while *GetMatchingProductForId* is for everything else. I've updated `MWSProducts.get_products()` to use *GetMatchingProduct* when the *id_type* is `"ASIN"`
